core/src/dunit/scala/io/snappydata/cluster/CassandraSnappyDUnitTest.scala

```diff
@@ -453,4 +454,13 @@ class CassandraSnappyDUnitTest(val s: String)
       case t: Throwable => assert(assertion = false, s"Unexpected exception $t")
     }
   }
+
+  def doTestSNAP3152(): Unit = {
+    logInfo("Running testSNAP3152")
+    submitAndWaitForCompletion("io.snappydata.cluster.jobs." +
+        "SNAP3152TestJob",
+      s"--conf spark.cassandra.connection.host=localhost" +
+          s" --conf connectorJar=$cassandraConnectorJarLoc")
+    logInfo("Job Completed Successfully")
+  }
```

Comment:
Could also check that the deployed jar is accessible from outside the job as well.